config/variables.js

```diff
@@ -199,8 +199,21 @@ const variables = {
     'screen-tablet': 768,
     'screen-sm-mobile': 400
   },
+  dimensions: {
+    'grid-base': '8px',
+    'spacing-base': '24px',
+    'homepage-max-width': '800px',
+    'site-max-width': '1200px',
+    'site-margins': '24px',
+    'site-margins-mobile': '16px',
+    'site-min-width': '320px'
+  },
```

Comment:
The dimensions object is another ACCESS OG holdover. Would like to avoid using it. The grid base is available as a top level variable configuration and can be access with the map get method in SASS; `map-get($variables, 'grid')`.